clients/venice-client/src/main/java/com/linkedin/venice/fastclient/DispatchingAvroGenericStoreClient.java

```diff
@@ -251,99 +202,71 @@ protected CompletableFuture<V> get(GetRequestContext requestContext, K key) thro
      *                       1. complete routeRequestFuture by passing in the status (200/404/etc) which will handle(decrement)
      *                          {@link InstanceHealthMonitor#pendingRequestCounterMap}
      *                       2. complete valueFuture by passing in either null/value/exception
-     * 4. transportFutures => List of transportFuture for this request (eg: 2 if speculative query is enabled or even more depends on
-     *                        health of the replicas).
+     *
      */
-    List<CompletableFuture<TransportClientResponse>> transportFutures = new LinkedList<>();
-    requestContext.requestSentTimestampNS = System.nanoTime();
-    for (String route: routes) {
```

Comment:
The loop was really confusing the first time when I looked at this. Good riddance.

Reply:
yeah, it was for speculative query support mainly.